src/link/rgblink.1

```diff
@@ -70,9 +77,11 @@ to fix these so that the program will actually run in a Game Boy:
 .D1 $ rgbfix -v bar.gb
 .Sh SEE ALSO
 .Xr rgbasm 1 ,
+.Xr rgblink 5 ,
 .Xr rgbfix 1 ,
 .Xr rgbds 7
 .Sh HISTORY
 .Nm
 was originally written by Carsten S\(/orensen as part of the ASMotor package,
-and was later packaged in RGBDS by Justin Lloyd.
+and was later packaged in RGBDS by Justin Lloyd. It is now maintained by a
+number of contributors on GitHub.
```

Comment:
![screenshot from 2017-04-01 11-27-13](https://cloud.githubusercontent.com/assets/6632664/24577879/39826b7e-16ce-11e7-8a31-e0bb38334ee7.png)
If I leave the dot it places it after the URL.

I'm not sure about how it looks, anyway.